docs/rule/no-autofocus-attribute.md

```diff
@@ -0,0 +1,31 @@
+# no-autofocus-attribute
+
+✅ The `extends: 'recommended'` property in a configuration file enables this rule.
```

Comment:
We should remove this line (we can't include a new rule in `recommended` until a major version bump release).